src/main/kotlin/org/ozwillo/dcimporter/service/ConnectorsService.kt

```diff
@@ -81,6 +81,27 @@ class ConnectorsService(private val businessAppConfigurationRepository: Business
             }
     }
 
+    fun clone(id: String, siret: String): Mono<HttpStatus>{
```

Comment:
a service should not return a HTTP status

Reply:
Now it return a Mono<BusinessAppConfiguration> or a Mono.error(ConflictException).